net-irc/hexchat/hexchat-2.12.1.ebuild

```diff
@@ -0,0 +1,142 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+inherit fdo-mime gnome2-utils mono-env multilib python-single-r1
+
+DESCRIPTION="Graphical IRC client based on XChat"
+HOMEPAGE="https://hexchat.github.io/"
+
+if [[ "${PV}" == "9999" ]] ; then
+	inherit git-r3
+	SRC_URI=""
+	EGIT_REPO_URI="git://github.com/hexchat/hexchat.git"
+else
+	SRC_URI="https://dl.hexchat.net/hexchat/${P}.tar.xz"
+	KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux"
+fi
+
+LICENSE="GPL-2 plugin-fishlim? ( MIT )"
+SLOT="0"
+IUSE="dbus debug +gtk libcanberra libnotify libproxy libressl lua nls perl +plugins plugin-checksum plugin-doat plugin-fishlim plugin-sysinfo python spell ssl theme-manager"
+REQUIRED_USE="
+	python? ( plugins ${PYTHON_REQUIRED_USE} )
+	perl? ( plugins )
+	lua? ( plugins )
+
+	plugin-checksum? ( plugins )
+	plugin-doat? ( plugins )
+	plugin-fishlim? ( plugins )
+	plugin-sysinfo? ( plugins )
+"
+
+COMMON_DEPEND="dev-libs/glib:2
+	dbus? ( sys-apps/dbus )
+	gtk? ( x11-libs/gtk+:3 )
```

Comment:
+1
If hexchat still supports gtk+-2.x I want a USE flag to switch between gtk+-2 and gtk+-3